# if you get an error about cv2 not being found then you can run this command
# it will copy the "shared object" file into your local environment path
# you can also verify that this is in your system path by printing the Python sys.path variable
cp /usr/lib/python3.6/dist-packages/cv2.cpython-36m-aarch64-linux-gnu.so /home/donkey/env/lib/python3.6

